### DISCUSSION TITLE
Support for additional NOSs

### DISCUSSION THREAD

**Discussion by tsmjay** [napalm-automation/napalm#1995] [Ideas]
Hi,

Are there any plans to add support for NOSs like DENT or SONiC?

This could be done initially through a community driver.

I believe Netmiko supports Linux and Dell SONiC device types.

Thanks.

**Comment by ktbyers** [comment]
@tsmjay Community drivers are driven by having someone that wants to write a driver for a platform (as well as maintain it going forward). So if you are interested in doing that, let us know.


Regards, Kirk

**Comment by tsmjay** [comment]
Hi @ktbyers,

Is the statement about Netmiko correct?

Is there a roadmap for NAPALM features?

I notice there is a gNMI community driver repository, which could potentially be used with SONiC.

Tom
